pkg/gofr/handler.go

```diff
@@ -70,6 +72,11 @@ func (h handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		// Execute the handler function
 		result, err = h.function(c)
 
+		// Log the error if any in the format (traceID errorMessage) with the color code 202
+		if err != nil {
+			h.container.Logger.Errorf("\u001B[38;5;8m%s \u001B[38;5;%dm%s\u001B[0m ", traceID, logging.DEBUG, err.Error())
```

Comment:
I have few points to mention here:
- why are we using logging.DEBUG for color code (in comment we have mentioned 202)
- this is implementation of prettyPrint method, which is only applicable for terminal logs
- in this case it will also reflect in json logs, and will appear like:
```{"level":"ERROR","time":"2024-08-06T11:23:09.338264+05:30","message":{"trace_id":"e5ad35bb304bc5b85ee97c87a551752a","span_id":"fe3a1121994dfed3","start_time":"2024-08-06T11:23:09.337988+05:30","response_time":274,"method":"GET","user_agent":"PostmanRuntime/7.36.3","ip":"[::1]:49877","uri":"/error","response":500},"gofrVersion":"dev"}
{"level":"ERROR","time":"2024-08-06T11:23:25.13114+05:30","message":"\u001b[38;5;8m95582574451ce935fcacd030a98557f0 \u001b[38;5;202msome error occurred\u001b[0m ","gofrVersion":"dev"}
```

@aditya028 - Can you please take a look at this.

Reply:
@srijan-27 can you review the changes